rules/no-array-instanceof.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const create = context => ({
+	BinaryExpression: node => {
+		if (node.operator === 'instanceof' && node.right.type === 'Identifier' && node.right.name === 'Array') {
+			if (node.left.type === 'Identifier') {
```

Comment:
Yup. I just realized that i dont need to explicitly check for `ArrayExpression` or an identifier. All i need is if i have `instanceof Array`. Thank you.